lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -683,6 +702,53 @@ def test_grading_failure(self, error_message, mock_grades_iter, _mock_current_ta
             }
         ])
 
+    @patch('lms.djangoapps.instructor_task.tasks_helper.runner._get_current_task')
```

Comment:
Have you thought about making the pre-existing`self.student_2` an inactive enrollee?  By doing so, you wouldn't need a new test case here.